requirements/edx/base.txt

```diff
@@ -17,10 +17,10 @@ defusedxml==0.4.1
 django-babel-underscore==0.5.1
 markey==0.8  # From django-babel-underscore
 django-celery==3.2.1
-django-config-models==0.1.3
+django-config-models==0.1.8
 django-countries==4.0
 django-extensions==1.5.9
-django-filter==0.11.0
+django-filter==1.0.4
```

Comment:
This is related to DRF's deprecation of [`DjangoFilterBackend`](http://www.django-rest-framework.org/topics/3.5-announcement/#djangofilterbackend). The functionality has been moved to the `django_filters` package.

@doctoryes did you do a pass to make sure we're importing `DjangoFilterBackend` and `FilterSet` from `django_filters` instead of `rest_framework`? Also, something that bit me when doing this upgrade for another service: DRF's browseable APIs will break unless you include `django_filters` in INSTALLED_APPS. For more, see https://github.com/edx/credentials/pull/284.